### PR TITLE
Remove usage of outdated pyschemes dependency

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
## Description
*This PR adds/fixes ...*

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for clinical-db):
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] go clinical-stage.scilifelab.se
- [ ] submit a correct order

### Expected test outcome
- [ ] check that the order is processed without errors
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in changelog
- [ ] Deploy this branch
- [ ] Inform to ES

### CODE REVIEW

File: cg/meta/orders/api.py [hunk 1]
```diff
@@ -40,10 +40,6 @@ def submit(self, project: OrderType, order_in: OrderIn, user_name: str, user_mai
 
         Main entry point for the class towards interfaces that implements it.
         """
-        try:
-            ORDER_SCHEMES[project].validate(order_in.dict())
```
Comment (moonso) [inline cg/meta/orders/api.py:44]: Should this not be validated no more?

Comment (patrikgrenfeldt) [reply cg/meta/orders/api.py:44]: hey MM!  I don't know, do you thinks it adds enough to be justified?

Comment (moonso) [reply cg/meta/orders/api.py:44]: Hi @patrikgrenfeldt !
If we know they are validated in the front end I don't think they need to be validated here. However if they are not I think we should replace the pyschemas with pydantic models

**Comment by patrikgrenfeldt** [comment]
Pydanic validation implemented here: https://github.com/Clinical-Genomics/cg/pull/1313

**Pull request closed without merging**